presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergMetadata.java

```diff
@@ -275,8 +277,10 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
         String tableName = schemaTableName.getTableName();
 
         Schema schema = new Schema(toIceberg(tableMetadata.getColumns()));
```

Comment:
Let's change `toIceberg` to return `Schema` and move this logic inside there. This will avoid any future callers of this method from having the same problem.